tests/rustdoc/enum/enum-variant-non_exhaustive.type-alias-code.html

```diff
@@ -0,0 +1 @@
+<code>pub type TypeAlias = <a class="enum" href="enum.Type.html" title="enum foo::Type">Type</a>;</code>
```

Comment:
It's not what we want to check. It's the declaration of the aliased `pub enum` and in particular to ensure that its variants have the attributes displayed.